lexer/lexer_test.go

```diff
@@ -0,0 +1,11 @@
+package lexer
+
+import (
+	"testing"
+)
+
+// NOTE:
+// Read the test JSON files and returns whether the files are valid JSON or not
+func TestNextToken(t *testing.T) {
+	test := "{}[]:,"
```

Comment:
Add some tests for a valid JSON file in the test data folder. Currently, the only JSON token that will pass the test is `{},:[]`, which is fine, data types have yet to be implemented

Reply:
Completed. Testing a JSON test file